low-level/sensors/impl/generic/node_sas_port.py

```diff
@@ -285,35 +313,27 @@ def check_and_send_alert(self):
         # Update sas ports status
         self.update_sas_ports_status()
 
-        if self.sas_phy_stored_alert == None:
-            # No info is stored for this node in Consul
+        # Check the version of stored alert
+        version = None
+        try:
+            # Try to get the version
+            # Exception will be raised if stored alert is None or no Version is available
+            version = self.sas_phy_stored_alert['version']
+        except:
+            logger.warning(f"Found no data or old data format for SASPortSensor, \
```

Comment:
anywhere version is getting updated anywhere still, so control will go to this exception everytime, even after this build, where version v1 would be updated.
About data versioning,  i meant to update in path /var/cortx/sspl/data/, to have /var/cortx/sspl/v1/data/ through filestore class change. It would basically touch all classes which are using this currently. And any old data outside v1 folder(old builds) should be read & converted to required version format(if changed) and move to under v1 folder.

 Address this in future. We had actually done this earlier, not sure when this got reverted ! have v1 in data folder path